step-api/src/test/java/org/jenkinsci/plugins/workflow/structs/DescribableHelperTest.java

```diff
@@ -461,6 +463,17 @@ public E getE() {
         }
     }
 
+    @Issue("JENKINS-26619")
+    @Test public void getterDescribableList() throws Exception {
+        roundTrip(GitSCM.class, map(
+            "extensions", Arrays.asList(map("$class", CleanBeforeCheckout.class.getSimpleName())),
```

Comment:
Before the fix, this map value was a singleton list of an actual `CleanBeforeCheckout` instance.